### PR TITLE
Camera crash fix

### PR DESCRIPTION
**Pull request by Adictya**
Related to [#87](https://github.com/AgoraIO-Community/app-builder-core/pull/87)

### CODE REVIEW

File: src/Rtc/Create.tsx [hunk 1]
```diff
@@ -51,16 +51,28 @@ const Create = ({
...
           });
           dispatch({
             type: 'LocalMuteVideo',
-            value: [ToggleState.disabled],
+            value: [ToggleState.enabled],
           });
-          console.error('No devices', e);
+        } catch (e) {
+          if (!e.status?.audioError) {
+            dispatch({
```
Comment (nitte93) [inline src/Rtc/Create.tsx:64]: Please verify for native devices as well.

**Review by nitte93 — CHANGES_REQUESTED** [review]

File: src/Rtc/Create.tsx [hunk 2]
```diff
@@ -167,18 +167,41 @@ const Create = ({
...
-                value: [ToggleState.disabled],
+                value: [ToggleState.enabled],
               });
               dispatch({
                 type: 'LocalMuteVideo',
-                value: [ToggleState.disabled],
+                value: [ToggleState.enabled],
               });
+            } catch (e) {
+              if ((e as any).status) {
```
Comment (nitte93) [inline src/Rtc/Create.tsx:180]: destructure the error object for better readability. 

 `const {status} = error`
 
 `const {audioError, videoError} = status`
 
 Then have the respective checks for audio and video undefined and their respective logic.

**Review by nitte93 — CHANGES_REQUESTED** [review]

File: src/Rtc/Create.tsx (hunk 2 above)
Comment (Adictya) [reply src/Rtc/Create.tsx]: Solved in the latest commit

**Comment by nitte93** [comment]
@EkaanshArora can you take some time out to review this PR ?

**Comment by EkaanshArora** [comment]
@nitte93 good to mergenow

**Review by nitte93 — APPROVED** [review]

**Pull request merged**